### PR TITLE
(RE-6447) Build ruby against openssl 1.0.2f

### PR DESCRIPTION
**Pull request by underscorgan**
This is to address CVE-2016-0701

### CODE REVIEW

**Comment by underscorgan** [comment]
Built puppet with this and it worked.

File: ruby/include/ruby-1.9.1/i386-mingw32/ruby/config.h [hunk 1]
```diff
@@ -109,7 +109,7 @@
 #define HAVE_INT64_T 1
 #define SIZEOF_INT64_T 8
 #define HAVE_UINT64_T 1
-#define SIZEOF_UINT64_T 8
```
Comment (stahnma) [inline ruby/include/ruby-1.9.1/i386-mingw32/ruby/config.h]: @underscorgan was this the actual fix?

File: ruby/include/ruby-1.9.1/i386-mingw32/ruby/config.h [hunk 2]
```diff
@@ -109,7 +109,7 @@
...
+#define SIZEOF_UINT64_T 0
```
Comment (shrug) [inline ruby/include/ruby-1.9.1/i386-mingw32/ruby/config.h]: this is an interesting change

Comment (melissa) [inline ruby/include/ruby-1.9.1/i386-mingw32/ruby/config.h]: hmmm, this shouldn't have changed....

Comment (underscorgan) [inline ruby/include/ruby-1.9.1/i386-mingw32/ruby/config.h]: I have no idea why this line changed, I grabbed the build from jenkins :|

**Comment by shrug** [comment]
:+1:

**Pull request merged**